### PR TITLE
chore: add a few comments and small code fixes in addTransmissibilityCouplingPattern for poromechanics with conforming fractures

### PR DESCRIPTION
**Pull request by ryar9534**
Adds a missing const and changes a globalIndex to a localIndex, as well as using flow solver discretization name to get stencil.

As it turns out, there was no indexing error.

### CODE REVIEW

**Review by paveltomin — APPROVED** [review]

**Comment by paveltomin** [comment]
@ryar9534 is it good to merge?

**Comment by ryar9534** [comment]
> @ryar9534 is it good to merge?

It should be - Matteo and I were considering waiting and including it in the solution to the import bug once we found it (just to avoid rebaselining multiple times) but lets just go ahead and do it. I will handle the rebaseline when the tests complete

**Review by rrsettgast — APPROVED** [review]

**Comment by rrsettgast** [comment]
@ryar9534 This doesn't seem to be passing tests.

**Review by jhuang2601 — APPROVED** [review]

File: src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsConformingFractures.cpp [hunk 1]
```diff
@@ -371,7 +371,7 @@ addTransmissibilityCouplingPattern( DomainPartition const & domain,
     // Get the finite volume method used to compute the stabilization
     NumericalMethodsManager const & numericalMethodManager = domain.getNumericalMethodManager();
     FiniteVolumeManager const & fvManager = numericalMethodManager.getFiniteVolumeManager();
-    FluxApproximationBase const & stabilizationMethod = fvManager.getFluxApproximation( this->solidMechanicsSolver()->getStabilizationName() );
+    FluxApproximationBase const & stabilizationMethod = fvManager.getFluxApproximation( this->flowSolver()->getDiscretizationName() );
```
Comment (CusiniM) [inline src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsConformingFractures.cpp]:
```suggestion
    FluxApproximationBase const & fvDiscretization = fvManager.getFluxApproximation( this->flowSolver()->getDiscretizationName() );
```

**Pull request merged**